test.py

```diff
@@ -20,14 +19,19 @@
 
 while True:
     dynamic_objects = []
-    reference_points = []
     timestamp = time.time()
+    reference_point = None
 
     # Processing frames
     for camera in cameras:
         camera.update_frame()
         dynamic_objects.extend(object_detection.get_dynamic_objects(camera))
-        reference_points.extend(vision_processing.ReferencePoint.from_apriltags(camera))
+        reference_point = vision_processing.ReferencePoint.from_apriltags(camera)
```

Comment:
Why is this code duplicated in the test?

Reply:
Yes, this is very redundant and needs to be refactored. My current thought is to put this logic in a class, and have either different instances or child classes, for testing on desktop and running on google coral hardware.

Reply:
Done!